src/credential/sd_jwt_vc.rs

```diff
@@ -0,0 +1,130 @@
+use base64::prelude::*;
+use ssi::{
+    claims::{
+        sd_jwt::{SdJwtBuf, RevealedSdJwt},
+        jwt::{AnyClaims, JWTClaims},
+        vc::{v1::Credential as _, v2::Credential as _},
+        JwsString,
+    },
+    prelude::AnyJsonCredential,
+};
+use uuid::Uuid;
+
+use crate::{CredentialType, KeyAlias};
+
+use super::VcdmVersion;
+
+#[derive(uniffi::Object, Debug, Clone)]
+/// A verifiable credential secured as a JWT.
+pub struct SdJwtVc {
+    id: Uuid,
+    sd_jwt: JwsString,
+    credential: AnyJsonCredential,
+    credential_string: String,
+    header_json_string: String,
+    payload_json_string: String,
+    disclosures: Option<Vec<String>>,
+    key_alias: Option<KeyAlias>,
+}
+
+#[uniffi::export]
+impl SdJwtVc {
+    /// The VdcCollection ID for this credential.
+    pub fn id(&self) -> Uuid {
+        self.id
+    }
+
+    /// The version of the Verifiable Credential Data Model that this credential conforms to.
```

Comment:
Please always include "W3C" when you mean that, just given how overloaded the terms have gotten.

It's also not clear that all SD-JWT-VCs are indeed W3C VCs, or that SD-JWT can be a VCDM V1 (do you see otherwise?). You may need to add a non-conformant result value.